app/ledger.js

```diff
@@ -110,7 +110,24 @@ let notificationTimeout = null
 const doAction = (action) => {
   var i, publisher
 
+/* TBD: handle
+
```

Comment:
yes, at some time in the future, we're not going to count idle when you are on an audio(/video) page... so, i've just put in a comment as a reminder that a check can start there.